### DISCUSSION TITLE
HmIP-MOD-TM Lüfterstellung Button fehlt

### DISCUSSION THREAD

**Discussion by Silv0rShad0w** [SukramJ/aiohomematic#1919] [Q&A]
Servus an alle,

Ich habe das ganze auf raspberry matic laufen und die Integration auf meinem Home Assistant installiert.

Ich habe die Torsteuerung Hm-IP-TM installiert. Mir fehlt in Home Assistant aber der Button für die Lüfterstellung?
Button für die LED ist da. Auch für Auf und Zu uns Stop ist der Button da. Allerdings gibt es auch noch einen Button für Lüftersteuerung, in raspberry matic ist der zu sehen uns auswählbar, nur in der Integration sehe ich ihn nicht.

Hat wer das selbe Problem oder es schon lösen können?

**Comment by SukramJ** [comment]
Da für gibt's keinen Button. Schieb den slider zwischen 10-50% dann ist das Gerät in der Vent Position.
Dafür kannst Du dir dann selber einen Button bauen.

**Comment by Silv0rShad0w** [comment]
Oh, tatsächlich, man lernt nie aus.

Hast du da evtl nen snippet für den button?

**Reply by SukramJ** [reply]
Nein, bitte schau inder HA Doku nach.

mit der Aktion cover.set_position stellst Du das Tor ein .

**Comment by Silv0rShad0w** [comment]
danke dir trotzdem.

Ziel ist ohnehin dann eine Automation in Verbindung mit nem Humidity Sensor.
